docs/concepts/operator.md

```diff
@@ -11,6 +11,7 @@ A lot of production- and enterprise-grade features were built into Gardener for
 
 There is a [Helm chart](../../charts/gardener/operator) which can be used to deploy the `gardener-operator`.
 Once deployed and ready, you can create a `Garden` resource.
+Note that there can only be one `Garden` resource per system at a time.
```

Comment:
Shall we add a check for "one Garden" at the beginning of `Reconcile` method too and let the operator stop if there is more than one `Garden`? Just in case someone creates a second garden while the webhook config is gone for any reason.